noisemodelling-emission/src/main/java/org/noise_planet/noisemodelling/emission/railway/cnossos/RailWayCnossosParameters.java

```diff
@@ -67,14 +66,14 @@ public RailWayCnossosParameters() {
 
 
 
-    public RailWayCnossosParameters(double[] lWRolling, double[] lWTractionA, double[] lWTractionB, double[] lWAerodynamicA, double[] lWAerodynamicB, double[] lWBridge) {
+    /*public RailWayCnossosParameters(double[] lWRolling, double[] lWTractionA, double[] lWTractionB, double[] lWAerodynamicA, double[] lWAerodynamicB, double[] lWBridge) {
```

Comment:
If this function is never used and you want to remove it, delete the code not comment it out.
Except for Java Getter/Setter, don't remove the functions even if it is not used (will be used outside of the library)